openmp/runtime/src/kmp_tasking.cpp

```diff
@@ -5505,7 +5506,7 @@ void __kmp_print_tdg_dot(kmp_tdg_info_t *tdg, kmp_int32 gtid) {
   KA_TRACE(10, ("__kmp_print_tdg_dot(exit): T#%d tdg_id=%d \n", gtid, tdg_id));
 }
 
-// __kmp_start_record: launch the execution of a previous
+// __kmp_exec_tdg: launch the execution of a previous
```

Comment:
A NFC change.

Reply:
That should go in a separate PR. LLVM only allows squash commits when merging a PR.

As for coding style, we don't really have a strict standard for `libomp`. I'd recommend following the older code style. By "older," I mean the code from around 2017, not the stuff added in the last year or two.

Reply:
Got it 😊 Thanks for the heads-up!